tensorflow/python/client/timeline.py

```diff
@@ -618,19 +669,26 @@ def analyze_step_stats(self, show_dataflow=True, show_memory=True):
         chrome_trace=self._chrome_trace,
         allocator_maximums=self._allocator_maximums)
 
-  def generate_chrome_trace_format(self, show_dataflow=True, show_memory=False):
+  def generate_chrome_trace_format(self, show_dataflow=True, show_memory=False, op_time="schedule"):
     """Produces a trace in Chrome Trace Format.
 
     Args:
       show_dataflow: (Optional.) If True, add flow events to the trace
         connecting producers and consumers of tensors.
       show_memory: (Optional.) If True, add object snapshot events to the trace
         showing the sizes and lifetimes of tensors.
+      op_time: (Optional.) How the execution time of op is shown in timeline.
+        Possible values are "schedule", "gpu" and "all".
+        "schedule" will show op from the time it is scheduled to the end of the scheduling.
+          Notice by the end of its scheduling its async kernels may not start yet.
+          It is shown using the default value from step_stats.
+        "gpu" will show op with the execution time of its kernels.
+        "all" will show op from the start of its scheduling to the end of its last kernel.
```

Comment:
will show op with the execution time of its kernels on GPU.

Reply:
I have modified the comment. Thank you.